packages/runtime/src/templates/edge/runtime.ts

```diff
@@ -13,6 +13,12 @@ export interface FetchEventResult {
   waitUntil: Promise<any>
 }
 
+export interface I18NConfig {
```

Comment:
Can this type be imported directly from Next.js? https://github.com/vercel/next.js/blob/fdff88bb62fabb825f0cb68cba3101bb572a8888/packages/next/server/config-shared.ts#L21-L26

Reply:
This is an edge function here, so we can't import stuff from node modules